tests/observatory/platform/test_observatory_config.py

```diff
@@ -542,25 +597,43 @@ def test_local_schema_airflow_variables(self):
         expected_errors = [{"airflow_variables": [{1: ["must be of string type"], "key1": ["must be of string type"]}]}]
         self.assert_sub_schema_valid(valid_docs, invalid_docs, schema, schema_key, expected_errors)
 
-    def test_local_schema_dags_projects(self):
+    def test_local_schema_workflows_projects(self):
         schema = make_schema(BackendType.local)
-        schema_key = "dags_projects"
+        schema_key = "workflows_projects"
 
         valid_docs = [
             {},
             {
-                "dags_projects": [
+                "workflows_projects": [
+                    {
+                        "package_name": "academic-observatory-workflows",
+                        "package": "/path/to/academic-observatory-workflows",
+                        "package_type": "editable",
+                        "dags_module": "academic_observatory_workflows.dags",
+                    },
+                    {
+                        "package_name": "oaebu-workflows",
+                        "package": "/path/to/oaebu-workflows/dist/oaeby-workflows.tar.gz",
```

Comment:
should this be oaebu-workflows.tar.gz (instead of oaeby)

Reply:
Ah yeah, fixed that now